### PR TITLE
Create BookController class

### PR DESCRIPTION
**Pull request by Dimagaa**
Create BookDto and CreateBookRequestDto classes
Create BookMapper and MapperConfig classes
Create EntityNotFoundException class
Add findById(Long id) method in BookRepository interface Add getById(Long id) method in BookService interface

### CODE REVIEW

File: src/main/java/com/app/onlinebookstore/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    @GetMapping(value = "/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.getById(id);
+    }
+
+    @PostMapping
+    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+        return bookService.save(bookDto);
```
Comment (Denis-Balako) [inline src/main/java/com/app/onlinebookstore/controller/BookController.java]: Don't lie to yourself 😄 
bookDto.class != BookDto.class
rename the variable

File: src/main/java/com/app/onlinebookstore/mapper/BookMapper.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com.app.onlinebookstore.config.MapperConfig;
+import com.app.onlinebookstore.dto.BookDto;
+import com.app.onlinebookstore.dto.CreateBookRequestDto;
+import com.app.onlinebookstore.model.Book;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface BookMapper {
+
```
Comment (Denis-Balako) [inline src/main/java/com/app/onlinebookstore/mapper/BookMapper.java]: Redundant line 👾

File: src/main/java/com/app/onlinebookstore/service/BookService.java [hunk 3]
```diff
@@ -1,10 +1,13 @@
 package com.app.onlinebookstore.service;
 
-import com.app.onlinebookstore.model.Book;
+import com.app.onlinebookstore.dto.BookDto;
+import com.app.onlinebookstore.dto.CreateBookRequestDto;
 import java.util.List;
 
 public interface BookService {
-    Book save(Book book);
+    BookDto save(CreateBookRequestDto book);
```
Comment (Denis-Balako) [inline src/main/java/com/app/onlinebookstore/service/BookService.java]:
```suggestion
    BookDto save(CreateBookRequestDto requestDto);
```

**Review by Denis-Balako — APPROVED** [review]
Great job! Found some minor issues, not critical.

**Review by Vasul-Pankiv — APPROVED** [review]

**Pull request merged**